src/components/common/spinner.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+function SvgSpinner() {
+  return (
```

Comment:
Let's use arrow function syntax for a better view
```suggestion
const SvgSpinner = () =>(
```